### PR TITLE
grammar fixes

### PR DESCRIPTION
**Pull request by clairemuller**
Going through the docs and fixing small errors I'm finding along the way.

### CODE REVIEW

**Review by benarent — APPROVED** [review]
Ty @clairemuller . Keep the PRs coming!

File: docs/4.2/quickstart.md [hunk 1]
```diff
@@ -120,25 +121,25 @@ NOTE: Make sure grav-00:3080 points at a Teleport proxy which users can access.
...
 open this URL in a web browser to complete the registration process.
 
 ## Step 4: Register a User
 
-* If the machine where you ran these commands has a web browser installed you
+* If the machine where you ran these commands has a web browser installed, you
 should be able to open the URL and connect to Teleport Proxy right away.
 
-* If the are working on a remote machine you may need to access the Teleport
+* If the are working on a remote machine, you may need to access the Teleport
```
Comment (webvictim) [inline docs/4.2/quickstart.md]: `If the are` -> `If you are`

**Review by webvictim — APPROVED** [review]
:rocket:

File: docs/4.2/quickstart.md (hunk 1 above)
Comment (clairemuller) [reply docs/4.2/quickstart.md]: Good catch!

**Review by benarent — APPROVED** [review]

**Comment by russjones** [comment]
@benarent Can we merge this?

**Pull request merged**